<ui version="4.0" >
 <class>QGPSMainWindowUI</class>
 <widget class="QWidget" name="QGPSMainWindowUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="lblFixStatus" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFixTime" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpPosition" >
     <property name="title" >
      <string>Position</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Latitude</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLatitude" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" >
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Longitude</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLongitude" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" >
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Altitude</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtAltitude" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Course</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" >
        <item>
         <widget class="QLabel" name="textLabel4" >
          <property name="text" >
           <string>Speed</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSpeed" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" >
        <item>
         <widget class="QLabel" name="textLabel5" >
          <property name="text" >
           <string>Heading</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtHeading" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9" >
        <item>
         <widget class="QLabel" name="textLabel6" >
          <property name="text" >
           <string>Magnetic Variation</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtVariation" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox4" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="title" >
      <string>Satellites</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QGPSSatelliteTracker" native="1" name="satTracker" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <item>
             <widget class="QLabel" name="textLabel1_2" >
              <property name="text" >
               <string>Active Satellites</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtNumSats" >
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel2_2" >
              <property name="text" >
               <string>Fix Type</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtFixType" >
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>132</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="SatelliteStrengthView" native="1" name="StrengthView" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGPSSatelliteTracker</class>
   <extends>QWidget</extends>
   <header>qgpssatellitetracker.h</header>
  </customwidget>
  <customwidget>
   <class>SatelliteStrengthView</class>
   <extends>QWidget</extends>
   <header>SatelliteStrengthView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
